packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -135,4 +142,101 @@ export default class ScheduledPaymentModule {
 
     return { txs: [transaction, setGuardTransaction], expectedModuleAddress };
   }
+
+  async estimateExecutionGas(
+    moduleAddress: string,
+    tokenAddress: string,
+    amount: string,
+    payeeAddress: string,
+    fee: Fee,
+    maxGasPrice: string,
+    gasTokenAddress: string,
+    salt: string,
+    payAt: number,
+    gasPrice: string
+  ): Promise<number>;
+  async estimateExecutionGas(
+    moduleAddress: string,
+    tokenAddress: string,
+    amount: string,
+    payeeAddress: string,
+    fee: Fee,
+    maxGasPrice: string,
+    gasTokenAddress: string,
+    salt: string,
+    recursDayOfMonth: number,
```

Comment:
Can we name this `recurringDayOfMonth` to be consistent with the namings in from the RFC?

Reply:
renamed